src/services/f1_fantasy_service.py

```diff
@@ -11,6 +11,8 @@
 from core.race import Race, RaceStatus
 from http_client import HTTPClient, HTTPMethod
 
+LEAGUE_ID = "14886408"
```

Comment:
can we move this param into an environment variable?

Reply:
I prefer to not do it right now, since it's not clear how and when to use it. After the 2nd race, I will understand better